smarc_bt/src/auv_config.py

```diff
@@ -35,7 +35,7 @@ def __init__(self):
         self.YAW_PID_ENABLE_TOPIC = 'ctrl/dynamic_heading/pid_enable'
         self.DEPTH_PID_ENABLE_TOPIC = 'ctrl/dynamic_depth/pid_enable'
         self.VEL_PID_ENABLE_TOPIC = 'ctrl/dynamic_velocity/pid_enable'
-
+        self.HEARTBEAT_TOPIC = 'sam_bt/heartbeat'
```

Comment:
Did you check with Nils that this is the topic to publish the heartbeat?

Reply:
Hello Özer, I check with Nils, it should be lolo/heartbeat, and everything should be in the lolo namespace now. Do you know how to change the namespace? Is that creating a new bt_lolo.launch changing all sam -> lol in launch file work? Does the heartbeat also need for sam? If now, do we want a new lolo_bt.py?

Reply:
You can achieve the proper topic similar to this: 
`self.START_STOP_DVL_NAMESPACE = "/"+self.robot_name+'/core/start_stop_dvl'`
so heartbeat would be `"/"+self.robot_name+"/heartbeat".`

These values are just defaults, the script that launches the mission will set the namespaces and robot name and pass it to us here, so we do not need a separate launch file at this level. 
I don't think SAM _needs_ a heartbeat, but it is not a _problem_ to have it either. So again, no need to separate anything at this point.

Reply:
There is a new commit. The topic name is 'heartbeat', so depending on the robot name, it will be lolo/heartbeat (as expected) / sam/heartbeat